asset/js/setup.js

```diff
@@ -33,10 +33,17 @@ function setupAnchors() {
   });
 }
 
+let userAfterSetup = () => {};
```

Comment:
`userAfterSetup` has a vague meaning, how about changing it to `executeAfterSetupScripts`?